tests/acceptance/v1.controller.acceptance.ts

```diff
@@ -1982,5 +1982,38 @@ describe('V1 controller (acceptance)', () => {
       expect(response.body).to.have.property('error')
       expect(response.body.error.message).to.startWith('Restricted endpoint: contract address not allowed')
     })
+
+    it('invokes GET /v1/{appId} and successfully relays a request only through the altruist', async () => {
```

Comment:
Thank you for adding the test.
This should be `invokes POST`

Reply:
it does, is a typo, fixed